website/sidebars.json

```diff
@@ -87,7 +87,6 @@
         "type": "subcategory",
```

Comment:
IMO it would make sense to put all of the SQL pages together in the ToC: either in the same subcategory, or in different subcategories that are adjacent.

Reply:
🤦🏼‍♂️ You're right.